docs/components/board/_index.md

```diff
@@ -24,9 +24,14 @@ The [RDK](/internals/rdk) also provides the [`GPIOPin` interface](#gpiopin-api)
 
 ## Configuration
 
-Configure a *board* component on your robot to communicate with the other [components](/components) of the robot, signaling as overseen by a computer running `viam-server`.
+Configure a *board* component on your robot to communicate with the other [components](/components) of the robot.
+Signaling is overseen by a computer running `viam-server`.
 
-Supported board models are single-board computers (SBCs) with GPIO pins and CPUs capable of running `viam-server`, GPIO peripheral devices that must connect to external computers, or PWM peripheral devices that must connect to CPUs providing GPIO pins.
+Supported board models use various architectures:
```

Comment:
Not sure if it's ideal to use architecture in this way when we have a lot of other stuff about architecture in the more traditional programming sense-- unfortunately I don't have a better idea though 😆 TBH I don't know if splitting this out into bullets is ideal either as the model information has more on this as well. Maybe it's more key info for users than I'm thinking though? Really your call

Reply:
I think the bullets help a lot because the original paragraph is hard to parse, and the bullets quickly show that there are 3 possible scenarios. Instead of calling them "architectures," do you think it would make sense to replace this sentence with
```suggestion
A board can be:
```
and then change the bullets to be singular (i.e. "A single-board computer....", "A GPIO peripheral device..." etc?